main.cpp

```diff
@@ -1 +1,239 @@
+#include <iostream>
+#include <string>
+#include <cstdlib>
+#include <ctime>
+#include <vector>
+#include <charconv>
+#include <algorithm>
+#include <iterator>
 
+using namespace std;
+
+void print(int guesses = 0) // print the hangman
+{ 
+    if (guesses >= 1) {cout << "  | ";}
+    if (guesses >= 2) {cout << "\n  | ";}
+    if (guesses >= 3) {cout << "\n  O ";}
+    if (guesses == 4) {cout << "\n /";}
+    if (guesses == 5) {cout << "\n /|";}
+    if (guesses >= 6) {cout << "\n /|\\";}
+    if (guesses == 7) {cout << "\n / ";}
+    if (guesses >= 8) {cout << "\n / \\";}
+    if (guesses >= 9) {cout << "\n ___";}
+    if (guesses == 10) {cout << "\n|   |\nYOU LOST!";}
+}
+
+void printoptions() // print options
+{
+    cout << "\nSelect One The Following Option:"
+         << "\n[1] Guess a letter"
+         << "\n[2] Guess the word"
+         << "\n[3] Check guessed letters"
+         << "\n[4] View options again"
+         << "\n[5] Quit game" << endl
+         << "[6] Check Current Word (FOR TESTING ONLY)\n";
+}
+
+void printword(const string word, const vector<char> cg) // function to print
+{
+    int i;
+    for (i = 0; i < word.length(); ++i)
+    {
+        if (count(cg.begin(), cg.end(), word[i]))
+        {
+            std::cout << word[i] << " ";
+        }
+        else
+        {
+            cout << "_ ";
+        }
+    }
+}
+
+int main()
+{
+    char input;                                                                                            // for user's letter guesses
+    int choice;                                                                                            // for user choice
+    string wordinput;                                                                                      // for when user wants to guess the entire word
+    vector<char> guessedchar;                                                                              // array for all the letters the user have guessed
+    vector<string> list{"special", "random", "roast", "firetruck", "throw", "trend", "blast", "gasoline"}; // list of possible questions
+    vector<char> correctguesses;                                                                           // letters that were guessed correctly
+    int guesses;                                                                                           // number of guesses by the user
+    srand(time(NULL));
+    int random = rand() % list.size(); // randomise the word
+    string word = list[random];
+    printoptions(); // to print the options
+    // Start Code
+    while (guesses <= 10)
```

Comment:
This '10' number is still not a variable, and it's used multiple times.
Use a #define (look it up) or a const variable for it